tests/clients/digitalocean/test_digitaloceanclient.py

```diff
@@ -0,0 +1,206 @@
+import os
+import gevent
+import pytest
+import string
+from unittest import TestCase
+from jumpscale.loader import j
+
+
+class DigitalOcean(TestCase):
+    def setUp(self):
+        self.ssh_client_name = j.data.idgenerator.nfromchoices(10, string.ascii_letters)
+        self.dg_client_name = j.data.idgenerator.nfromchoices(10, string.ascii_letters)
+        ssh = j.clients.sshkey.get(name=self.ssh_client_name)
+
+        self.dg = j.clients.digitalocean.get(self.dg_client_name)
+        if os.getenv("SSH_PRIVATE_KEY_PATH") and os.getenv("DIGITAL_OCEAN_ACCESS_TOKEN"):
```

Comment:
Instead of getting env vars twice, we can get it once in the variables and then check the variables.